### DISCUSSION TITLE
Search Results Webpart arbitrary properties for templates

### DISCUSSION THREAD

**Discussion by poneilcsgp** [microsoft-search/pnp-modern-search#2175] [Q&A]
Is there any way to pass arbitrary properties to a template similar to passing the slot properties?  For example, I have a template that I'd like to reuse, but for some instances of the web part I'd like to show a button and others I'd like to hide it.    If it was possible to, say, add a slot variable called "ShowButton" and set it to true, 1, etc, then I could use a handlebars expression to show/hide that element.

**Comment by wobba** [comment]
See https://github.com/microsoft-search/pnp-modern-search/pull/2167 which is in the dev branch
